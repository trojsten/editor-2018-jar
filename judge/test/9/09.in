VECTOR_INT:
lode
72
19
45
68
60
9
47
17
26
42
29
50
67
34
58
66
10
65
40
13
43
63
15
24
21
1
44
41
22
70
36
57
32
39
7
64
23
38
5
72
35
30
54
56
61
31
52
33
20
27
11
16
14
51
3
49
53
2
59
37
46
8
28
71
55
6
48
12
25
4
18
62
69
INT:
nepouzijestentointnadarmo
0
