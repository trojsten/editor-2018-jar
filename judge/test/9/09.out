{"lode": [1, 21, 24, 15, 63, 43, 13, 40, 65, 10, 66, 58, 34, 67, 50, 29, 42, 26, 17, 47, 9, 60, 68, 45, 19, 69, 62, 18, 4, 25, 12, 48, 6, 55, 71, 28, 8, 46, 37, 59, 2, 53, 49, 3, 51, 14, 16, 11, 27, 20, 33, 52, 31, 61, 56, 54, 30, 35, 72, 5, 38, 23, 64, 7, 39, 32, 57, 36, 70, 22, 41, 44]}